plugins/woocommerce/client/legacy/css/twenty-nineteen.scss

```diff
@@ -1445,3 +1445,16 @@ form.checkout_coupon {
 		@include checkout-inline-error-message();
 	}
 }
+
+/**
+ * Log in form
+ */
+.woocommerce-page {
+	form {
+		.password-input {
+			input[type="text"] {
+				padding-right: 2.5rem;
```

Comment:
When changing the password visibility on Twenty Seventeen/Nineteen themes, the right padding needs to be adjusted.